manifest.json

```diff
@@ -28,18 +28,17 @@
       "format": "title",
       "name": "enrich_info",
       "title": "Enrichment",
-      "description": "The Enrichment API lets you look up person and company data based on an email or domain"
+      "description": "Transform the email address or domain in a person or company profile with more than 85 data points."
```

Comment:
Romain, the title is "Enrichment", so we really shouldn't use the word "enrich" to explain "enrichment".